### ISSUE TITLE
Not working in angular 2.3.1

### ISSUE THREAD

**Issue by jaronwright** [astronaut1712/ng2-medium-editor#2]
I could not get this working in my project. I have tried the suggestions in the other post any other idea's? Would it be possible to update this for 2.3.1?

**Comment by jkrajinovic** [comment]
Yes please. Trying to import with system.js but no success. Maybe some instructions or working example. ng v2.3.0. Thanks.
